ner_v2/api.py

```diff
@@ -687,6 +688,14 @@ def text(request):
             response = {"success": False, "error": str(err)}
             ner_logger.exception(response)
             return JsonResponse(response, status=400)
+        except es_exceptions.ConnectionTimeout as err:
```

Comment:
The `raise KeyError` from inside of the datastore will still hit the block at line 683, still returning 400 as status code because currently there is no way to distinguish between invalid request body and datastore error

Reply:
checking